public/managehashes.php

```diff
@@ -44,23 +46,62 @@
 <?php RenderToolbar($user, $permissions); ?>
 <script>
   function UpdateHashDetails(user, hash) {
+    $('#warning').html('Status: updating...');
     var name = $.trim($('#HASH_' + hash + '_Name').val());
     var labels = $.trim($('#HASH_' + hash + '_Labels').val());
     var posting = $.post('/request/game/modify.php', { u: user, g: <?php echo $gameID ?>, f: 4, v: hash, n: name, l: labels });
-    posting.done(onUpdateComplete);
+    posting.done(function onUpdateComplete(data) {
+        if (data !== 'OK') {
+            $('#warning').html('Status: Errors...' + data);
+        } else {
+            // Get comment date
+            var date = new Date();
+            var dateStr = date.getUTCDate() + ' ' + shortMonths[date.getUTCMonth()] + ' ' +  date.getUTCFullYear() + '<br>' + date.getUTCHours() + ':' + ('0' + date.getUTCMinutes()).slice(-2);
+
+            // Place comment on correct row depending on if the user has submitted a comment already
+            if ($('#commentTextarea').length) {
+                $('#feed tr:last').before('<tr class="feed_comment localuser system"><td class="smalldate">' + dateStr + '</td><td class="iconscommentsingle"></td><td class="commenttext">' + hash + ' updated by ' + user + '. Description: "' + name + '". Label: "' + labels + '"</td></tr>');
+            } else {
+                $('#feed').append('<tr class="feed_comment localuser system"><td class="smalldate">' + dateStr + '</td><td class="iconscommentsingle"></td><td class="commenttext">' + hash + ' updated by ' + user + '. Description: "' + name + '". Label: "' + labels + '"</td></tr>');
+            }
 
-    $('#warning').html('Status: updating...');
-  }
-
-  function onUpdateComplete(data) {
-    //alert( data );
-    if (data !== 'OK') {
-        $('#warning').html('Status: Errors...' + data);
-        //alert( data );
-    } else {
-        $('#warning').html('Status: OK!');
+            $('#warning').html('Status: OK!');
+        }
+    })
+}
+
+function UnlinkHash(user, gameID, hash, elem) {
+    if (confirm('Are you sure you want to unlink the hash ' + hash + '?'))
+    {
+        $('#warning').html('Status: updating...');
+        var posting = $.post('/request/game/modify.php', { u: user, g: gameID, f: 3, v: hash });
+        posting.done(function onUnlinkComplete(data) {
+            if (data !== 'OK') {
+                $('#warning').html('Status: Errors...' + data);
+            } else {
```

Comment:
See above - no need for `else` here.

Reply:
Guess I could've done the suggestion here where it would've worked 🤦

Reply:
Fixed.